### DISCUSSION TITLE
How to enable "Start Call" | Audio & Video Call Option

### DISCUSSION THREAD

**Discussion by jasonsan9364** [LiveHelperChat/livehelperchat#2041] [Q&A]
I have enabled the feature of "Audio & Video Call Option" and it's working fine at the operator end. However, during testing, I was not able to make the same option enabled for my customer. Please see the screenshot below for your reference and comments. 

![start-call error](https://github.com/LiveHelperChat/livehelperchat/assets/156453430/7cf98fb2-7db2-493e-bb42-ee3ce2646fe1)

**Comment by remdex** [comment]
Screenshot tells absolute nothing except that icon is not loading :) so debug or hire developer to do it...

**Reply by jasonsan9364** [reply]
Here is a video link for your review and comments. 

[https://screenrec.com/share/pvHLKD8qGF]
